src/compas_fab/ghpython/path_planning.py

```diff
@@ -6,10 +6,7 @@
 from timeit import default_timer as timer
 
 import compas
-from compas.datastructures import Mesh
```

Comment:
I'm so looking forward for the time when we can get rid of this file. I believe it's only used for vrep planning examples.